catroid/src/main/java/org/catrobat/catroid/formulaeditor/InternFormulaParser.java

```diff
@@ -217,7 +217,7 @@ private FormulaElement term() throws InternFormulaParserException {
 
 			case BRACKET_OPEN:
 				getNextToken();
-				currentElement.replaceElement(new FormulaElement(FormulaElement.ElementType.BRACKET, null, null, null,
+				currentElement.replaceElement(new FormulaElement(FormulaElement.ElementType.BRACKET, "", null, null,
```

Comment:
this would be nicer if youd fix this by checking for null in stringWithoutTrailingZero, to make call to "getValue" save for all kinds of tokens.